### PR TITLE
Adjust wealth input to individual level.

### LINKED ISSUE #724: BUG: input variable `vermögen_bedürft_bg` uses endogeneous grouping
### Bug description

- we request the input variable `vermögen_bedürft_bg` however this grouping is endogeneous. Also, it is used for `hh`-wide transfers like Wohngeld which seems incorrect if `bg` and `hh` differ. Probably should be `vermögen_bedürft_hh` (which we somehow need to distribute on Bedarfsgemeinschaften) or we request individual-level wealth?

See also #412

### PR DESCRIPTION
**Pull request by ChristianZimpelmann**
Closes #724

### CODE REVIEW

File: docs/gettsim_objects/means_testing.md [hunk 1]
```diff
@@ -4,14 +4,19 @@
...
 
-'vermögen_bedürft_bg'. Despite small differences over the transfers, we decided, for
-now, to require only one wealth variable as input and use it for all transfers.
+'vermögen_bedürft'. Despite small differences over the transfers, we decided, for now,
+to require only one wealth variable as input and use it for all transfers.
+
+Note that GETTSIM requests wealth on the individual level as input. This variable is
+then aggregated on the respective grouping level for each transfer. In most data sets
+wealth is only measured on the household level and, hence, needs to be split up on the
+individual level before feeding it into GETTSIM.
```
Comment (hmgaudecker) [inline docs/gettsim_objects/means_testing.md:19]: Great. Please add that this is due to the fact that transfers are calculated at the sub-household level (Bedarfsgemeinschaften), which is endogenous. And that if in doubt, robustness checks regarding different distributions of wealth across household members might be in order.

File: src/_gettsim_tests/test_data/benefit_checks/2006/hh_id_4.yaml
Comment (hmgaudecker) [inline src/_gettsim_tests/test_data/benefit_checks/2006/hh_id_4.yaml:1]: I think I would prefer to revert the changes in `benefit_checks`. These are not about getting the bedarfsgemeinschaft right, purely about testing what happens once we have wealth at that level. We do use "intermediate" variables all the time for that and should continue to do so here IMO.

**Review by hmgaudecker — COMMENTED** [review]
Excellent, thanks!!! Quick reactions after a glance, won't have time until the afternoon and thus leaving it to @MImmesberger to approve & merge!

**Comment by hmgaudecker** [comment]
Closes #412

File: src/_gettsim_tests/test_data/benefit_checks/2006/hh_id_4.yaml
Comment (ChristianZimpelmann) [reply src/_gettsim_tests/test_data/benefit_checks/2006/hh_id_4.yaml:1]: As from this PR on, Wohngeld and ALG 2 use different wealth variables (on `hh` and `eg` level) we need to adjust the files anyhow. The alternative would be to include both `vermögen_bedürft_eg` and `vermögen_bedürft_hh`. Additionally, Kinderzuschlag will most likely require wealth on the individual level anyhow in the future (see #412).

I would rather stick to the implemented changes.

Comment (hmgaudecker) [reply src/_gettsim_tests/test_data/benefit_checks/2006/hh_id_4.yaml:1]: Fine, I did not realise the tests were doing all checks at the same time. Thanks!

**Review by MImmesberger — APPROVED** [review]
Looks good to me, thanks!
